compose.yml

```diff
@@ -235,12 +242,20 @@ services:
     healthcheck:
       test: ["CMD-SHELL", "curl --fail --silent --write-out '{\"status\":%{http_code}}' --output /dev/null http://127.0.0.1:8080/health"]
       start_period: 10s
-      interval: 60s
+      interval: 10s
       timeout: 10s
       retries: 10
     depends_on:
-      - db
+      db:
+        condition: "service_healthy"
+      localstack:
+        condition: "service_healthy"
     environment:
       ASPNETCORE_ENVIRONMENT: Production
       ConnectionStrings__EvDatabase: 'Server=db;Database=cdp;Username=cdp_user;Password=cdp123;'
-      Sqs__ServiceURL: "http://localstack:4566"
\ No newline at end of file
+      Sqs__ServiceURL: "http://localstack:4566"
+      Sqs__AuthenticationRegion: "eu-west-1"
+      InboundQueue__Name: "ev-inbound"
+      InboundQueue__MaxNumberOfMessages: 10
+      InboundQueue__WaitTimeSeconds: 20
+      OutboundQueue__Name: "ev-outbound"
```

Comment:
Only `Sqs__ServiceURL` is necessary here, as this the only different setting between appsettings.Development.json and docker.